lib/workload/stateful/statefulStackCollectionClass.ts

```diff
@@ -0,0 +1,47 @@
+import { Construct } from 'constructs';
+import { Stack, Environment, StackProps } from 'aws-cdk-lib';
+
+import { SharedStack, SharedStackProps } from './stacks/shared/stack';
+import { TokenServiceProps, TokenServiceStack } from './stacks/token-service/deploy/stack';
+import { IcaEventPipeStack, IcaEventPipeStackProps } from './stacks/ica-event-pipe/stack';
+
+export interface StatefulStackCollectionProps {
+  sharedStackProps: SharedStackProps;
+  tokenServiceStackProps: TokenServiceProps;
+  icaEventPipeStackProps: IcaEventPipeStackProps;
+}
+
+export class StatefulStackCollection {
```

Comment:
Oh, the "collection" class. I like this naming.! Good one, Will. 👍

Reply:
Initially was set to "manager" class, but I think we had enough managers in OrcaBus 🤔